lib/pbkit/pbkit.c

```diff
@@ -2365,6 +2365,23 @@ void pb_push4to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD p
     *(p+4)=param4;
 }
 
+void pb_push4fto(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3, DWORD param4)
```

Comment:
Argument types wrong in commit `pbkit: Add pb_push4fto function `

Reply:
Great find, thanks!
Fixed.

Reply:
CI actually had an error because of this. I didn't have to look very hard.